core/src/stmt.rs

```diff
@@ -28,6 +28,11 @@ pub trait MongoStatement: Debug {
     fn execute(&mut self, _: &MongoConnection) -> Result<bool> {
         Err(Error::UnsupportedOperation("execute"))
     }
+    // Closes the cursor.
+    // Only MongoQuery supports this workflow. The other statements don't.
+    fn close_cursor(&mut self) -> Result<()> {
+        Err(Error::UnsupportedOperation("close_cursor"))
```

Comment:
That sounds good. I started implementing it with Ok(()) but decided to go with an Error to mirror `execute`.

I'm going to make this change, but I'm going to go a bit further and not return anything since there is no way for this to return an error _other than_ the current default implementation. That will mean the default implementation is truly a no-op.